MAUI/Scheduler/getting-started.md

```diff
@@ -602,4 +602,6 @@ this.Content = scheduler;
 
 N> It is not applicable if the `View` is `Timeline Month` and it is applied only when the `ShowWeekNumber` property is `enabled.`
 
-![customize-the-week-number-text-style-in-maui-scheduler](images/getting-started/customize-the-week-number-text-style-in-maui-scheduler.png)
\ No newline at end of file
+![customize-the-week-number-text-style-in-maui-scheduler](images/getting-started/customize-the-week-number-text-style-in-maui-scheduler.png)
+
+N> You can refer to our [.NET MAUI Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Scheduler Example](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main) that shows you how to render the Heatmap Chart in Vue.
```

Comment:
Scheduler in .NET MAUI
![image](https://github.com/syncfusion-content/maui-docs/assets/95272441/81e1724b-b33f-4322-919e-2ac450311ec3)